### PR TITLE
fixed requests after setting  db timezone

### PR DESCRIPTION
**Pull request by adlbrtbsr**

### CODE REVIEW

File: backend/movies/views.py [hunk 1]
```diff
@@ -102,16 +102,19 @@ def get(self, request, format=None):
         date = request.GET.get('date', None)
 
         if date:
-            parsed_date = parser.parse(date, dayfirst=True)
+            parsed_date = parser.parse(date)
             movie_night = MovieNight.objects.get(night_date__date=parsed_date)
 
-            print(movie_night)
+            if movie_night.selected_movie is not None:
```
Comment (LissaGreense) [inline backend/movies/views.py:108]: if movie_night.selected_movie: is enough :)

File: backend/movies/views.py [hunk 2]
```diff
@@ -47,6 +47,8 @@ def get(self, request, format=None):
 
         for movie in all_movies:
             average_rating = Rate.objects.filter(movie=movie).aggregate(Avg('rating'))['rating__avg']
+            if average_rating is None:
```
Comment (LissaGreense) [inline backend/movies/views.py:50]:
```
>>> imNone = None
>>> imNotNone = True
>>> imNotNoneToo = ['']
>>> imNotNoneTooToo = {'yey':1}
>>> imNotNoneTooTooToo = object()
>>> if imNone:
...     print("ImNone passed")
... 
>>> if imNotNone:
...     print("imNotNone passed")
... 
imNotNone passed
>>> if not imNotNone:
...     print("not imNotNone passed")
... 
>>> if imNotNoneToo:
...     print("imNotNoneToo passed")
... 
imNotNoneToo passed
>>> if not imNotNoneToo:
...     print("not imNotNoneToo passed")
... 
>>> if not imNotNoneTooToo:
...     print("not imNotNoneTooToo passed")
... 
>>> if imNotNoneTooToo:
...     print("imNotNoneTooToo passed")
... 
imNotNoneTooToo passed
>>> if imNotNoneTooTooToo:
...     print("imNotNoneTooTooToo passed")
... 
imNotNoneTooTooToo passed
>>> if not imNotNoneTooTooToo:
...     print("not imNotNoneTooTooToo passed")
... 

```

Again there is no need to compare every var to None. Better to write it shorter and do not over-complicate logic in ifs ;)

File: backend/movies/views.py (hunk 1 above)
Comment (adlbrtbsr) [reply backend/movies/views.py]: ok

File: backend/movies/views.py (hunk 2 above)
Comment (adlbrtbsr) [reply backend/movies/views.py]: tl;dr

Comment (LissaGreense) [reply backend/movies/views.py]: Do not read it, type it in your python console and get some understanding ;)

Comment (LissaGreense) [reply backend/movies/views.py]: You even removed continue clause, gj ;)

**Review by LissaGreense — APPROVED** [review]
### Yey! Nice fixes

**Pull request merged**